includes/class-sensei-teacher.php

```diff
@@ -935,6 +937,8 @@ public function notify_admin_teacher_course_creation( $new_status, $old_status,
 
         // Send mail
         Sensei()->emails->send( $recipient, $subject , Sensei()->emails->get_content( $template ) );
+				
+				do_action('sensei_after_sending_email');
```

Comment:
ah, wait. Sorry, this is 'sensei_after_sending_email'. Yes, this one you can move into send().

We cannot touch 'sensei_before_mail' - it must stay before gettext functions.